docs/contributor/01-00-installation.md

```diff
@@ -2,8 +2,6 @@
 - [Install API Gateway](#install-api-gateway)
```

Comment:
```suggestion
```
We can remove the whole table of contents. I think it makes no sense to keep it when we have only two sections there